src/helm/benchmark/run_expander.py

```diff
@@ -819,6 +819,57 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
             )
             for value in self.values
         ]
+    
+    
+class ChatMLRunExpander(RunExpander):
+    
+    name = "chatml"
+    
+    def __init__(self):
+        self.name = type(self).name
+
+    def expand(self, run_spec: RunSpec) -> List[RunSpec]:
+        adapter_spec = run_spec.adapter_spec
+        # according to https://github.com/openai/openai-python/blob/main/chatml.md#few-shot-prompting
+        # few-shot examples should do `<|im_start|>system name=example_user` or `<|im_start|>system name=example_assistant`
+        # but it is also possible to put examples into a user message.
+        if run_spec.name.split(':')[0] in ('msmarco',):
```

Comment:
There's a bunch of duplicate code in the three cases...the only difference seems to be `output_prefix`, so could you just set that in the if/else, and then create `adapter_spec` outside?

Reply:
Sure!